tests/users/test_views.py

```diff
@@ -0,0 +1,110 @@
+import pytest
+from django.core import mail
+from django.urls import reverse
+from django_webtest import DjangoTestApp
+
+from vitrina.users.models import User
+
+
+credentials_error = "Neteisingi prisijungimo duomenys"
+name_error = "Vardas negali būti trumpesnis nei 3 simboliai, negali turėti skaičių"
+terms_error = "Turite sutikti su naudojimo sąlygomis"
+reset_error = "Naudotojas su tokiu el. pašto adresu neegzistuoja"
+
+
+@pytest.fixture
+def user():
+    user = User.objects.create_user(email="test@test.com", password="test123")
+    return user
+
+
+@pytest.mark.django_db
+def test_login_with_wrong_credentials(csrf_exempt_django_app: DjangoTestApp, user: User):
+    resp = csrf_exempt_django_app.post(reverse('login'), {
+        'email': "test@test.com",
+        'password': "wrongpassword"
+    })
```

Comment:
It is better to use a webtest form posting, this way, you won't need csrf excempt and more things will be tested, for example actual form rendering in templates will be tested, making sure if all the fields are actually in the form.

Could you also switch to webtest form handling in all other tests in this file?

https://docs.pylonsproject.org/projects/webtest/en/latest/forms.html